Lib/uuid.py

```diff
@@ -201,8 +203,30 @@ def __init__(self, hex=None, bytes=None, bytes_le=None, fields=None,
             # Set the version number.
             int &= ~(0xf000 << 64)
             int |= version << 76
-        self.__dict__['int'] = int
-        self.__dict__['is_safe'] = is_safe
+        object.__setattr__(self, 'int', int)
+        object.__setattr__(self, 'is_safe', is_safe)
+
+    def __getstate__(self):
+        d = {attr: getattr(self, attr) for attr in self.__slots__}
+        # is_safe is a SafeUUID instance.  Return just its value, so that
```

Comment:
Is the value portable? Seems there is a problem in 3.7 which should be fixed (this is a different issue).

Reply:
I'm not sure what you mean. The value is one of `None`, `0` or `-1`, so it should be "portable" for any meaning of that word.

We have to keep just the value to avoid errors when unpickling in Python versions <3.7, where the `SafeUUID` enum isn't defined. Using the `.value`, `is_safe` is ignored and unpickling succeeds.